cmd/pdcli/alerts/cmd.go

```diff
@@ -32,71 +36,110 @@ import (
 )
 
 type Alert struct {
-	IncidentID string
-	Name       string
-	ClusterID  string
-	Severity   string
-	Status     string
+	IncidentID  string
+	AlertID     string
+	ClusterID   string
+	ClusterName string
+	Name        string
+	Console     string
+	Labels      string
+	LastCheckIn string
+	Severity    string
+	Status      string
+	Sop         string
+	Token       string
+	Tags        string
 }
 
-var args struct {
-	high       bool
-	low        bool
-	assignment string
-	columns    string
+var options struct {
+	high        bool
+	low         bool
+	assignment  string
+	columns     string
+	interactive bool
+	incidentID  bool
 }
 
 var Cmd = &cobra.Command{
 	Use:   "alerts",
 	Short: "This command will list all the open high alerts assigned to self.",
+	Args:  cobra.MaximumNArgs(1),
 	RunE:  alertsHandler,
 }
 
 func init() {
 
 	// Urgency
-	Cmd.Flags().BoolVar(&args.low, "low", false, "View all low alerts")
-	Cmd.Flags().BoolVar(&args.high, "high", true, "View all high alerts")
+	Cmd.Flags().BoolVar(&options.low, "low", false, "View all low alerts")
+	Cmd.Flags().BoolVar(&options.high, "high", true, "View all high alerts")
 
 	// Incident Assignment
 	Cmd.Flags().StringVar(
-		&args.assignment,
+		&options.assignment,
 		"assigned-to",
 		"self",
 		"Filter alerts based on user or team",
```

Comment:
any example of user or team you can mention will be helpful

Reply:
The same is mentioned in the ReadMe too, wouldn't that do?
If `assigned-to=team`, it defaults to Platform-SRE for now, will change it in the teams ticket.